codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpProtocolServiceClient.kt

```diff
@@ -50,20 +48,40 @@ class AWSHttpProtocolServiceClient(
                     ConfigProperty("authSchemes", SmithyHTTPAuthAPITypes.AuthSchemes.toOptional(), authSchemesDefaultProvider)
                 }
                 "retryStrategyOptions" -> {
-                    writer.addImport(SwiftDependency.SMITHY_RETRIES_API.target)
-                    ConfigProperty("retryStrategyOptions", SmithyRetriesAPITypes.RetryStrategyOptions, "AWSClientConfigDefaultsProvider.retryStrategyOptions()", true)
+                    ConfigProperty(
+                        "retryStrategyOptions",
+                        SmithyRetriesAPITypes.RetryStrategyOptions,
+                        { it.format("AWSClientConfigDefaultsProvider.retryStrategyOptions()") },
```

Comment:
Config properties now write their defaults with a block that allows a SwiftWriter to be passed in.  This allows symbols in the default to be written correctly.